stream-chat-android/src/main/java/com/getstream/sdk/chat/utils/DateFormatter.kt

```diff
@@ -0,0 +1,40 @@
+package com.getstream.sdk.chat.utils
+
+import android.annotation.SuppressLint
+import io.getstream.chat.core.internal.InternalStreamChatApi
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Date
+import java.util.Locale
+
+@InternalStreamChatApi
+public object DateFormatter {
+
+    @SuppressLint("ConstantLocale")
+    private val DEFAULT_LOCALE = Locale.getDefault()
+    private val DATE_FORMAT: DateFormat = SimpleDateFormat("MMM d", DEFAULT_LOCALE)
+    private val TIME_FORMAT: DateFormat = SimpleDateFormat("HH:mm", DEFAULT_LOCALE)
```

Comment:
In case the default `Locale` is `US`. Shouldn't we display time in AM/PM format?

Reply:
This is a feature request, this is meant to be purely a refactor PR 😄 

Jokes aside, yes, we might want to consider different locales at some point. This way it'll be easier to change to a new implementation if we have to.